src/Table/elements.js

```diff
@@ -63,29 +89,23 @@ export const THeader = styled.thead`
       fonts: { weight },
     },
   }) => weight.thick};
-  line-height: 1.8rem;
 
+  line-height: 1.8rem;
   text-transform: uppercase;
-
   color: ${({ theme: { palette } }) => palette.darkGrey};
 `;
 
 export const TableHeaderCell = styled.th`
   position: relative;
   height: 4.4rem;
-  padding: 0 1.2rem 0 2rem;
+  padding: 0 ${TABLE_CELL_PADDING_H}rem;
```

Comment:
I think this change causes this:
![Screenshot 2019-12-30 at 12 03 18](https://user-images.githubusercontent.com/6019103/71579333-68681c00-2afc-11ea-9925-dcc5d12af2fd.png)